packages/blade/jest.native.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     },
   },
   moduleFileExtensions: ['native.ts', 'native.tsx', 'ts', 'tsx', 'js', 'json', 'node'],
-  testMatch: ['**/__tests__/**/*.native.test.{ts,tsx}', '**/__tests__/**/*.test.{ts,tsx}'],
+  testMatch: ['**/*.native.test.{ts,tsx}', '**/*.test.{ts,tsx}'],
```

Comment:
The second glob pattern `**/*.test.{ts,tsx}` this would still match a test file for `web.ts`? For example `User.web.ts` wouldn't it? So it would end up running tests for `.web` as well.

Reply:
Nupe. I've explicitly ignored that. Here's [the magic](https://github.com/razorpay/blade/pull/271/files#diff-c44cf9f865bf8178b6ee129ff776089b04b4e9738e6c121eee474f329aaef0abR5)